contracts/container/contract.go

```diff
@@ -1721,6 +1722,66 @@ func OwnerOf(tokenID []byte) interop.Hash160 {
 	return owner[1 : 1+interop.Hash160Len]
 }
 
+// Tokens returns iterator over IDs of all existing containers.
+//
+// Tokens implements optional NEP-11 method.
+func Tokens() iterator.Iterator {
+	return storage.Find(storage.GetReadOnlyContext(), []byte{containerKeyPrefix}, storage.KeysOnly|storage.RemovePrefix)
+}
+
+// Properties returns properties of referenced container.
+//
+// The properties are:
+//
+//	name: Base58-encoded tokenID;
+//	owner: Base64-encoded [Info.Owner];
+//	nonce: Base64-encoded [Info.Nonce];
+//	basicACL: [Info.BasicACL];
+//	storagePolicy: Base64-encoded [Info.StoragePolicy];
```

Comment:
These two are somewhat questionable to me as well. Not that it's an error to return them, but we do create some potential naming conflict. We can try concentrating on attributes first.

Reply:
dropped for now